revscoring/languages/tests/test_bosnian.py

```diff
@@ -1,8 +1,11 @@
 import pickle
 
-from nose.tools import eq_
+from pytest import mark
 
-from .. import bosnian
+try:
+    from .. import bosnian
+except:
+    pytestmark = mark.nottravis
```

Comment:
What's going on here?

Reply:
That's a great question, importing bosnian causes error in travis because the the dictionary is not there and pytestmark = mark.nottravis is a global identifier asking pytest not to touch this file. We can go with mark.skip as well. Doesn't matter.

Reply:
Gotcha.  This is OK, but maybe we should have a comment there?  I'm not sure.  If this is going to be a common pattern, it shouldn't need a comment, but in this case we're only doing it twice.

Reply:
Done :)